sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuShuffleEnv.scala

```diff
@@ -109,14 +109,16 @@ object GpuShuffleEnv extends Logging {
       case rapidsShuffleManager: RapidsShuffleManagerLike =>
         rapidsShuffleManager.initialize
       case _ =>
-        val caseClassLoader =
-          classOf[RapidsShuffleManagerLike].getClassLoader
-        val rsmLoaderViaShuffleManager = shuffleManager.getClass.getSuperclass.getInterfaces.collectFirst {
+        val rsmLoaderViaShuffleManager = shuffleManager.getClass.getSuperclass.getInterfaces
+        .collectFirst {
           case c if c.getName == classOf[RapidsShuffleManagerLike].getName => c.getClassLoader
         }
```

Comment:
nit: Method calls are typically indented when starting on the new line
```suggestion
        val rsmLoaderViaShuffleManager = shuffleManager.getClass.getSuperclass.getInterfaces
          .collectFirst {
            case c if c.getName == classOf[RapidsShuffleManagerLike].getName => c.getClassLoader
          }
```